### DISCUSSION TITLE
prohd remote code 3

### DISCUSSION THREAD

**Discussion by f520k510** [GerryDazoo/Slinger#39] [Q&A] [answered]
Hello, 

Thanks for the great work.  I have the following problems.

So I set my remote code in the config.ini to 3 component for my prohd, but it won't work.
I tried RT but it only toggle between 0,1, and 2.  
I have no ways to know if code 3 will work or not.
I have a 500 connected with the same stb at component, or videosource 2, and code 2.
The remote for the 500 works great.

This makes me suspect that could the code 3 not being implemented?
Because the author of the slinger seems not have a prohd...
Just a wild guess.
Please help me.

**Comment by f520k510** [comment]
I uploaded the working bin file that was installed on component input to composite input.  In the code of remote I set code=1.
This time it works.  However, it is better to be able to use the remote on the code 3.
Thanks.

**Reply by Mike7766** [reply]
Is there step by step instructions on how to do this with slingbox

**Comment by GerryDazoo** [comment, marked as answer]
Based on some others experiences it is beginning to look like the remote code is the same as the VideoSource please try setting them both to the same number in your config.ini.   3 for your prohd and 2 for the 500.

 

Not going to three is a bug in my code that I’ll fix in the next release but the whole searching for the right code is now moot due to what I’ve learned from the softwares’ users.

 

*	Gerry

**Reply by f520k510** [reply]
Thanks for the reply.  I did try set the code=3 for my pro hd but it didn't work.  For now, I set it to code 1 and also uploaded bin to composite.  BTW, I know of a method to find the actual mapping code of each button to the custom remote.  If you have a prohd or solo, install slingplayer 2.0.4.  Open sbcore.ini located in C:\ProgramData\Sling Media\SlingPlayer\2.0 and add DebugRemote=1 under [main]   Then, when you hover over each button on the virtual remote, it will show the mapped key code.

Hope this helps.

**Comment by GerryDazoo** [comment]
I’ll give your little remote debug a try. I did not know this. This will be a big time saver for people with custom remotes.

I have no explanation for why code 3 doesn’t work.  Can you set your videosource to 3 and see what the software prints out for the remote key codes.

**Comment by f520k510** [comment]
Hi,
I tested again by setting remote code to 3 and this time it works.
I think maybe I didn't log on to watch video when I use the remote.
Sorry.

I hope the debug remote trick works for you.

**Comment by GerryDazoo** [comment]
Great, the remote truck worked as advertised, Will save a lot of people time getting their systems back on the air.

I’ll add that info to the release notes.
